examples/with-jest/index.test.js

```diff
@@ -0,0 +1,19 @@
+import {matcher, serializer} from 'jest-glamor-react'
+import React from 'react'
+import {shallow, render, mount} from 'enzyme'
+import {Wrapper, Title} from './index'
+
+expect.addSnapshotSerializer(serializer)
```

Comment:
Ah, actually, we should probably use: [`setupTestFrameworkScriptFile`](https://facebook.github.io/jest/docs/configuration.html#setuptestframeworkscriptfile-string)